File: c++/sort/bubbleSort.cpp
//Bubble Sort, O(n²)
#include <iostream>
using namespace std;

int main(){
	int array[] = {9,8,7,6,5,4,3,2,1};
	int length = sizeof(array)/sizeof(array[0]);
	for(int i=0; i<length; i++){
		for(int j=i+1; j<length; j++){
			if(array[j]<array[i]){
				swap(array[j], array[i]);
			}
		}
	}
	for(int k=0; k<length; k++){
		cout << array[k] << endl;
	}
}

File: python/sort/bubbleSort.py
#Bubble Sort, O(n²)

array = [9,8,7,6,5,4,3,2,1]

for i in range(len(array)):
    for j in range(i, len(array)):
        if array[j] < array[i]:
            array[i], array[j] = array[j], array[i]

print(array)
